scripts/bert/create_pretraining_data.py

```diff
@@ -216,7 +216,7 @@ def create_training_instances(x):
                             next_sentence_labels, segment_ids, valid_lengths
     """
     (input_files, tokenizer, max_seq_length, short_seq_prob,
-     masked_lm_prob, max_predictions_per_seq, vocab,
+     masked_lm_prob, max_predictions_per_seq, do_whole_word_mask, vocab,
```

Comment:
create_training_instances() function is also called in pretraining_utils.py , so the `do_whole_word_mask` parameter should be added there too.